SK.WebApp/src/services/consts.ts

```diff
@@ -0,0 +1,29 @@
+const apiAddress = "https://localhost:44324";
```

Comment:
Could you please make a structure as described in https://github.com/PolishDevCom/SocialKnow/pull/33#discussion_r595923520? If you have other idea, please share it here but you didn't say anything and you had already made legacy code 😄

Reply:
fixed